services/galley/src/Galley/API/Internal.hs

```diff
@@ -318,6 +332,14 @@ iPutTeamFeature ::
   Galley r (Public.TeamFeatureStatus a)
 iPutTeamFeature setter = Features.setFeatureStatus @a setter DontDoAuth
 
+iUnlockPayment ::
+  forall a r.
+  (Public.KnownTeamFeatureName a) =>
+  (TeamId -> Galley r Public.PaymentStatus) ->
```

Comment:
we probably don't need this.  it's only there in `getSelfDeletingMessagesInternal` etc. because of the different configuration types for different features.  payment status has no heterogeneous config attribute.